Tools/Firmware/CMakeLists.txt

```diff
@@ -7,6 +7,9 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
 set(ROOT_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../..)
 
+## TODO: Make this configurable via build script
+Set(MODEL_NAME, testmodel1)
```

Comment:
Looks like this should be pretty easy by passing in a `-DMODEL_NAME` value in the build script
https://cmake.org/cmake/help/latest/manual/cmake.1.html#cmdoption-cmake-D